Makefile.build

```diff
@@ -83,6 +83,17 @@ snapshots/hacl-c:
 	$(addprefix snapshots/kremlib/, kremlib.* testlib.* gcc_compat.h FStar.h Makefile)  \
 		./snapshots/hacl-c
 
+snapshots/nss: snapshots/snapshot-gcc-unrolled
+	# Compile everything but only copy out the things NSS is using.
+	mkdir -p snapshots/nss
+	cp snapshots/snapshot-gcc-unrolled/Curve25519.c snapshots/nss/hacl_curve25519_64.c
+	cp snapshots/snapshot-gcc-unrolled/Curve25519.h snapshots/nss/hacl_curve25519_64.h
+	cp snapshots/kremlib/FStar.h snapshots/nss/fstar_uint128.h
+	cp snapshots/kremlib/kremlib.h snapshots/nss
+	cp snapshots/kremlib/gcc_compat.h snapshots/nss
```

Comment:
In the current version of `kremlib` in NSS, `gcc_compat.h` is inlined into `kremlib.h`.
This will require either a patch to NSS or an inlining.

Reply:
I've reverted this change on kremlin master -- it's useful for some situations to only have gcc_compat.h

Reply:
This means we would have multiple unverified files instead of just one. I'll inline it mechanically  as `kremlib.h` is our TCB in HACL*.